public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -591,29 +584,27 @@ table(width="100%")
       code-example.
         &lt;tr ng-repeat="movie in movieList | orderBy : 'title'">
       :marked
-        Orders the collection as specified by the expression.
-        In this example, the movieList is ordered by the movie title.
+        Orders the collection, as specified by the expression.
+        In this example, the movie title orders the movieList.
     td
       :marked
         ### none
-        There is no comparable pipe in Angular&nbsp;2 for performance reasons.
-        Ordering/sorting the results should be coded in the component.
-        Consider building a custom pipe if the same ordering/sorting code
-        will be reused in several templates.
+        For performance reasons, no comparable pipe exists in Angular&nbsp;2.
+        Order/sort results in the component.
```

Comment:
Like similar wording before, this is hard to read (_results_ sounds like a verb instead of a noun). But I think this one is easier to fix:

Instead, use component code to order or sort results.